### ISSUE TITLE
Refactor JmxTreeWatcher to remove needless logging

### ISSUE THREAD

**Issue by benemon** [hawtio/hawtio#2521]
It's 2018. Do we really need hawt.io to log "Don't cha wish your console was hawt like me? ;-)" at startup?

**Comment by oscerd** [comment]
It's like that from the beginning. Just like a legacy.

**Comment by oscerd** [comment]
But let's vote to remove that, with the team. Personally it's not a problem

**Comment by benemon** [comment]
@oscerd yeah, I know. If something needs to happen there for user feedback, that's cool. It just feels a bit pointless, and a little bit grating when you fire up hawt.io and that gets spat out in my server logs every...damn...time...

I'm going to raise a PR with the change in, but a discussion would be good.

**Comment by gashcrumb** [comment]
> little bit grating when you fire up hawt.io and that gets spat out in my server logs every...damn...time...

Exactly, here's my +1 vote for removing it.
